source/extra/examples/minio-dev.yaml

```diff
@@ -0,0 +1,40 @@
+# Deploys a new Namespace for the MinIO Pod
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: minio-dev # Change this value if you want a different namespace name
+  labels:
+    name: minio-dev # Change this value to match metadata.name
+---
+# Deploys a new MinIO Pod into the metadata.namespace Kubernetes namespace
+#
+# The `spec.containers[0].args` contains the command run on the pod
+# The `/data` directory corresponds to the `spec.containers[0].volumeMounts[0].mountPath`
+# That mount path corresponds to a Kubernetes HostPath which binds `/data` to a local drive or volume on the worker node where the pod runs
+# 
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: minio
+  name: minio
+  namespace: minio-dev # Change this value to match the namespace metadata.name
+spec:
+  containers:
+  - name: minio
+    image: quay.io/minio/minio:latest
+    command:
+    - /bin/bash
+    - -c
+    args: 
+    - minio server /data --console-address :9001
+    volumeMounts:
+    - mountPath: /data
```

Comment:
Theres a possibility that this will work without any volumeMounts or volumes, using a purely ephemeral folder `/data` - but that feels too useless, as if the pod goes, all data and configs are lost. It would be the only quickstart we have where the data is not persistent. 

In K8s this adds some complexity unfortunately. But I am hoping users on K8s will be able to understand the intent here. If we get consistent feedback from users who are getting this wrong we can revise.